### DISCUSSION TITLE
How to make it more persistent?

### DISCUSSION THREAD

**Discussion by gerroon** [buggyj/tiddlyclip#81] [Q&A]
Hi,

It seems like I need to regularly do "dock here" in the TW tab while the browser (chromium) is open the whole time. I am wondering if there is a way to make it more peristent so that the extention does not forget which tab is TW?

thanks

**Comment by buggyj** [comment]
I am not understanding you question. 

I do not use chromium so do not know if there is a problem. 

With firefox and chrome, once a tiddlywiki is docked it remains dock until the tab is closed or reloaded.

**Reply by gerroon** [reply]
Hi,

That is not the experience I have here. I am unable to send snips to TW tab unless I "dock here" in the TW tab sometimes. I never close the TW tab and my browser is always open, so I am not sure why that would be the case.

Are there cases where the extention would loose connection beside closing the TW tab?

It sounds like once it is docked one should not be needing to redock it until the browser is restarted.

**Reply by buggyj** [reply]
There are two cases where the docking is lost:

1) If the tiddlywiki page is reload tiddlyclip will undock it. 

2) If the browser restarts the extension

I will try chromium

**Reply by buggyj** [reply]
I have tried with chromium on xubuntu but have not observed this error.

Here is a logger for pageloads and docks, unzip and drop it on you tiddlywiki in the browser. Reload and it will start logging - you will be able to see if the page reloads after docking.

[zbjenabledocklog.tid.zip](https://github.com/buggyj/tiddlyclip/files/10437482/zbjenabledocklog.tid.zip)

**Reply by buggyj** [reply]
You will find the log in the tiddler LogClip

**Reply by gerroon** [reply]
Thankks for investigating it, I will see why I have the issue. Would occasional network problems also cause that issue?
